lib/modules/userTagger.js

```diff
@@ -811,9 +811,15 @@ export async function applyTagToAuthor(thisAuthorObj: HTMLAnchorElement, noEdit:
 		);
 	}
 
+	let tagElement;
+	if (module.options.showTaggingIcon.value) {
+		tagElement = addAuthorTag(thisAuthorObj, noEdit, thisAuthor);
+	}
+
 	const tagObject = await getTagBatched(thisAuthor);
 
-	if (module.options.showTaggingIcon.value || !_.isEmpty(tagObject)) {
+	if (!_.isEmpty(tagObject)) {
+		if (tagElement && tagElement.parentElement) tagElement.parentElement.removeChild(tagElement);
```

Comment:
It's [part of the DOM 4 spec](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove), which also has a few other jQuery-like things (e.g. [closest()](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)).

We ship a separate version of jQuery because we can't use reddit's (due to sandboxing), so technically it would still work on profile pages. I plan to get rid of it eventually, though, so avoiding it is a good idea. (the main issue is we use a few [jQuery plugins](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/lib/vendor/jqueryPlugins.js) which would have to be replaced)